.copyrightIgnore

```diff
@@ -1,3 +1,3 @@
 //List of files that don't require copyrights
 //One file per line, and full paths must be used
-.copyright_whitelist
+.copyright_allowlist
```

Comment:
I think the file should be named `.copyrightignore` (lower-case) for consistency with other 'ignore' files and it should continue to be self-referential (the line should say `.copyrightignore`).